translations/authors/AUTHORS-de.md

```diff
@@ -0,0 +1,40 @@
+### Direktor der Wettbewerbe
+
+* Maria Kiskachi
+
+### Entwickler von Aufgaben
+
+* Alexander Belev
+* Alexander Kotov
+* Alexey Zakharov
+* Artur Liss
+* Vladimir Cherepanov
+* Maria of the Ninths
+* Mikhail Gnedashev
+* Mikhail Evdokimov
+* Roman Bykov
+
+### Разработчики проверяющей системы
```

Comment:
Entwickler des Verifizierungssystems?

Reply:
Oh my god,How Could I Have Missed It?

Okay let me correct all my mistakes.